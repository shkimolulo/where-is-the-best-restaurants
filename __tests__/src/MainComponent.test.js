import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

describe('MainComponent', () => {
    let MainComponent;
    
    beforeEach(() => {
        MainComponent = require('../../src/MainComponent').default;
    });

    it('렌더링 시킬 수 있다.', () => {
        // Given

        // When
        const json = renderer.create(<MainComponent />).toJSON()

        // Then
        expect(json).toMatchSnapshot();

    });
})