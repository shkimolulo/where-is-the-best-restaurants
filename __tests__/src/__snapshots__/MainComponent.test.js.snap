// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainComponent 렌더링 시킬 수 있다. 1`] = `
<View
  style={
    Object {
      "backgroundColor": "red",
      "flex": 1,
    }
  }
/>
`;
